source/event_loop.c

```diff
@@ -74,7 +87,7 @@ struct aws_event_loop *aws_event_loop_new_with_epoll(
  * retrieve the default type value.
  * If `aws_event_loop_override_default_type` has been called, return the override default type.
  */
-static enum aws_event_loop_type aws_event_loop_get_default_type(void) {
+enum aws_event_loop_type aws_event_loop_get_default_type(void) {
```

Comment:
This function returns a enum but the `#else` logs an error and the function does not return anything.